doc/standard_library.md

```diff
@@ -59,7 +59,7 @@ of each.
 - Reline ([GitHub][reline]): GNU Readline and Editline in a pure Ruby implementation
 - Resolv ([GitHub][resolv]): Thread-aware DNS resolver library in Ruby
 - SecureRandom ([GitHub][securerandom]): Interface for a secure random number generator
-- Set ([GitHub][set]): Provides a class to deal with collections of unordered, unique values
+- [Set](rdoc-ref:Set) ([GitHub][set]): Provides a class to deal with collections of unordered, unique values
```

Comment:
It should be able to use backquotes.

```suggestion
- `Set` ([GitHub][set]): Provides a class to deal with collections of unordered, unique values
```

Reply:
If it’s ok, I prefer using rdoc-ref for now as it has the missing reference check.